plugins/jobs/server/__init__.py

```diff
@@ -42,7 +46,18 @@ def scheduleLocal(event):
         fn = getattr(module, job.get('function', 'run'))
         fn(job)
 
+@access.token(scope=constants.REST_CREATE_JOB_TOKEN_SCOPE)
+def _authorizeRestJobCreation(event):
+    tokenModel = ModelImporter.model('token')
+    token = getCurrentToken()
+
+    if not tokenModel.hasScope(token, constants.REST_CREATE_JOB_TOKEN_SCOPE):
+        raise AccessException(
+            'Invalid token scope.\n'
+            'Required: %s.\n' % (constants.REST_CREATE_JOB_TOKEN_SCOPE))
```

Comment:
It wouldn't help with that particular case. I'm not actually sure how to engineer a solution for that at this time.